docs/workshop.md

```diff
@@ -0,0 +1,702 @@
+# Workshop
+
+This workshop uses Wegue version [`v1.0.0`](https://github.com/meggsimum/wegue/releases/tag/v0.25.0) but works for higher versions with no or little adaptation.
+
+## Prerequisites
+
+You should have basic knowledge of using a commandline and a basic understanding of HTML, CSS and JavaScript
```

Comment:
Maybe adding `Basic knowlegde of Vue.js helps but is not mandatory` would be good.

Reply:
done